Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1264,9 +1264,9 @@ Limitations
 
     * Cilium's eBPF kube-proxy replacement currently cannot be used with :ref:`gsg_encryption`.
     * Cilium's eBPF kube-proxy replacement relies upon the :ref:`host-services` feature
-      which uses eBPF cgroup hooks to implement the service translation. The getpeername(2)
-      hook address translation in eBPF is only available for v5.8 kernels. It is known to
-      currently not work with libceph deployments.
+      which uses eBPF cgroup hooks to implement the service translation. It is known to
+      currently not work with libceph deployments unless a v5.8 kernel with getpeername(2)
+      hook address translation in eBPF is available.
```

Comment:
We could maybe improve the sentence a little bit, this “it is known to currently not work” is hard to read, and your version also might make it unclear if kernel 5.8 is enough or if some other parameter needs to be set for `getpeername` to be available.

How about:

> Using it with libceph deployments currently requires support for the getpeername(2) hook address translation in eBPF, which is only available for kernels v5.8 and higher.

Reply:
Updated to follow your suggestion which to me looks much better. Can't comment too much on it since English is not my first language.

Reply:
Me neither ;) Thanks for the update!